src/PeraWalletConnect.ts

```diff
@@ -186,6 +187,24 @@ class PeraWalletConnect {
       throw new Error("PeraWalletConnect was not initialized correctly.");
     }
 
+    if (isMobile()) {
+      let peraWalletAppDeeplink;
+
+      try {
+        // This is to automatically open the wallet app when trying to sign with it.
+        peraWalletAppDeeplink = window.open(PERA_WALLET_APP_DEEP_LINK, "_blank");
```

Comment:
Let's use `generatePeraWalletAppDeepLink()` utility

```suggestion
        peraWalletAppDeeplink = window.open(generatePeraWalletAppDeepLink(), "_blank");
```